code/go/0chain.net/smartcontract/dbs/event/allocation.go

```diff
@@ -43,6 +43,8 @@ type Allocation struct {
 	SuccessfulChallenges     int64         `json:"successful_challenges"`
 	FailedChallenges         int64         `json:"failed_challenges"`
 	LatestClosedChallengeTxn string        `json:"latest_closed_challenge_txn"`
+
+	Users User `gorm:"foreignKey:Owner;references:UserID"`
```

Comment:
not sure if it's required?

Reply:
as per slack discussion, we need it right?